pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -195,9 +196,8 @@ func TestUpdateStatus(t *testing.T) {
 			Name: "foo",
 		},
 		Status: api.PersistentVolumeStatus{
-			Phase: api.VolumeBound,
-			// Set the same timestamp as original PV so this won't get updated on phase change breaking DeepEqual() later in test.
-			LastPhaseTransitionTime: pvStartTimestamp,
+			Phase:                   api.VolumeBound,
+			LastPhaseTransitionTime: &metav1.Time{Time: pvStartTimestamp.Time.Add(time.Second)},
```

Comment:
Why would we need to add exactly 1 second drift here? I don't see how this would help with the flakes.

Reply:
it is not related with this flake. it is used to test the new feature  which gate is on.